src/tests/Microsoft.PowerFx.Interpreter.Tests/RecalcEngineTests.cs

```diff
@@ -237,6 +238,22 @@ public void ChangeRecord()
             AssertUpdate("A-->12;");
         }
 
+        [Fact]
+        public void CheckFunctionCounts()

```

Comment:
>CheckFunctionCounts [](http://example.com/codeflow?start=20&length=19)

@lesaltzm / @bryancroteau-MSFT  - this is a bit wacky so I wanted to be explicit about it in a test. 

The problem is that we have defaults, and too many different things want different defaults and for it to happen automatically.